hs_core/views/__init__.py

```diff
@@ -2132,6 +2132,7 @@ def metadata_review(request, shortkey, action, uidb36=None, token=None, **kwargs
             f"This resource does not have a pending metadata review for you to { action }.",
         )
     else:
+        res.raccess.alter_review_pending_flags(initiating_review=False)
```

Comment:
this line is the real hotfix. all the rest is just optimizing a management command to add some additional helpful args